src/EcomProvider.cs

```diff
@@ -721,56 +715,101 @@ public EcomProvider()
+
+        mappings = GetMappingsByName(job.Mappings, "EcomGroups", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomManufacturers", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomVariantGroups", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomVariantsOptions", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomProducts", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomProductItems", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomProductsRelated", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomStockUnit", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomDetails", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomProductCategoryFieldValue", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomPrices", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomAssortments", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomAssortmentPermissions", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomAssortmentGroupRelations", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomAssortmentProductRelations", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomAssortmentShopRelations", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
+        mappings = GetMappingsByName(job.Mappings, "EcomVariantOptionsProductRelation", isSource);
+        if (mappings != null)
+            tables.AddRange(mappings);
+
         job.Mappings = tables;
     }
 
-    internal static IEnumerable<Mapping> GetMappingsByName(MappingCollection collection, string name)
+    internal static IEnumerable<Mapping> GetMappingsByName(MappingCollection collection, string name, bool isSource)
```

Comment:
looks like this can be non-static now?

Reply:
i am not sure what the benefits are for removing the "static" on it, and Visual Studio do say I should add it on them
![image](https://github.com/dynamicweb/DataIntegration.EcomProvider/assets/123721955/728c824e-a047-4934-9004-526a8c3b3880)

Reply:
since it is not exposed as public and it is not used as static then I think it is not needed to be static but if VS makes a warning because of that then just keep it as it is :)